subprojects/groovy-groovydoc/src/main/java/org/apache/groovy/antlr/GroovydocVisitor.java

```diff
@@ -78,7 +86,15 @@ protected SourceUnit getSourceUnit() {
 
     @Override
     public void visitClass(ClassNode node) {
-        List<String> imports = node.getModule().getImports().stream().map(ImportNode::getClassName).collect(Collectors.toList());
+        final Map<String, String> aliases = new HashMap<>();
+        final List<String> imports = new ArrayList<>();
+        for (ImportNode iNode : node.getModule().getImports()) {
+            String name = iNode.getClassName();
+            imports.add(name);
+            if (iNode.getAlias() != null && !iNode.getAlias().isEmpty()) {
+                aliases.put(iNode.getAlias(), name.replaceAll("\\.", "/"));
```

Comment:
`name.replaceAll("\\.", "/"))` can be replaced with `name.replace('.', '/'))` to avoid regex compilation

Reply:
Nice suggestion.